queries.sql

```diff
@@ -1,3 +1,29 @@
 /*Queries that provide answers to the questions from all projects.*/
 
```

Comment:
Please add the project title to the README file. A professional README should have a project title, description, and a set of instructions to run the project locally.